src/components/Controller/Controller.tsx

```diff
@@ -365,29 +374,29 @@ const Controller = () => {
                     <FontAwesomeIcon icon={faRotateLeft} />
                   </button>
                   <HotkeyButton
-                    label="Up"
+                    label={<FontAwesomeIcon icon={faArrowUp} />}
                     disabled={disableTransmitAction}
                     onClickFunction={() => write("button 4", autoUpdateFrame)}
                     className="h-16 w-16 bg-green-500"
                     shortcutKeys={"ArrowUp"}
                   />
                   <HotkeyButton
-                    label="Ok"
+                    label={<FontAwesomeIcon icon={faRightToBracket} />}

```

Comment:
Could we please change this icon to fa-check

Reply:
Done